FinSearchUnified/Tests/Bundle/StoreFrontBundle/Gateway/Findologic/CustomFacetGatewayTest.php

```diff
@@ -2,61 +2,59 @@
 
 namespace FinSearchUnified\Tests\Bundle\StoreFrontBundle\Gateway\Findologic;
 
-use Enlight_Controller_Front;
 use Enlight_Controller_Request_RequestHttp;
+use Enlight_Exception;
 use Exception;
 use FINDOLOGIC\Api\Responses\Response;
-use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder\NewQueryBuilder;
-use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder\NewQueryBuilderFactory;
+use FINDOLOGIC\Api\Responses\Xml21\Xml21Response;
+use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder\QueryBuilder;
+use FinSearchUnified\Bundle\SearchBundleFindologic\QueryBuilder\QueryBuilderFactory;
+use FinSearchUnified\Bundle\SearchBundleFindologic\ResponseParser\ResponseParser;
 use FinSearchUnified\Bundle\StoreFrontBundle\Gateway\Findologic\CustomFacetGateway;
 use FinSearchUnified\Bundle\StoreFrontBundle\Gateway\Findologic\Hydrator\CustomListingHydrator;
 use FinSearchUnified\Bundle\StoreFrontBundle\Struct\Search\CustomFacet;
+use FinSearchUnified\Tests\Helper\Utility;
 use FinSearchUnified\Tests\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
 use ReflectionException;
 use ReflectionObject;
 use Shopware\Bundle\SearchBundle\Facet\ProductAttributeFacet;
 use Shopware\Bundle\StoreFrontBundle\Service\ContextServiceInterface;
 use Shopware_Components_Config;
-use SimpleXMLElement;
+use Zend_Cache_Exception;
 
 class CustomFacetGatewayTest extends TestCase
 {
     protected function tearDown()
     {
         parent::tearDown();
 
-        Shopware()->Container()->reset('front');
-        Shopware()->Container()->load('front');
         Shopware()->Container()->reset('config');
         Shopware()->Container()->load('config');
-        Shopware()->Session()->offsetUnset('isSearchPage');
-        Shopware()->Session()->offsetUnset('isCategoryPage');
-        Shopware()->Session()->offsetUnset('findologicDI');
+
+        unset(
+            Shopware()->Session()->isSearchPage,
```

Comment:
Manual unsetting may also work, but internally they may have to cleanup some stuff in the session, so I do not feel confident in doing this, even if we may already do it at other places. To still make this clean, you may be able to chain the `offsetUnset` methods together, but I haven't checked that.